rust-runtime/aws-smithy-runtime/src/client/orchestrator.rs

```diff
@@ -125,15 +127,21 @@ async fn try_op(ctx: &mut InterceptorContext, cfg: &mut ConfigBag, interceptors:
     }
 
     loop {
+        if !ctx.rewind(cfg) {
+            break;
```

Comment:
Do we need to distinguish how we break out of the loop, i.e. due to a rewind failure vs. `ShouldAttempt::No`?

Reply:
It's a good question. I don't think so, but I can see that it'd be helpful for debugging purposes.